### PR TITLE
Jnugent/welcome page

### PR DESCRIPTION
**Pull request by jacknugent**
- Removed unused package.json dependencies
- Added bootstrap packages
- Replaced React boilerplate with CCA text/styles/icons
- Added typescript
- Added initial front-end form (will connect to backend in future PR)

Desktop (icon spins)
image

Mobile
image

A few changes from [the initial design](https://www.figma.com/file/PEEkfIkziGfLogDHxSex9K/CC-%2F%2F-Action-Tool?node-id=205%3A2):
- Added labels to form inputs. Makes the form more accessible.
- Made inputs larger and separated by two lines. Gives the form more space and also improves accessibility.

Open to feedback/suggestions

### CODE REVIEW

**Comment by kevincianfarini** [comment]
General comment (already communicated privately) -- I cannot build the frontend container. I plan on adding CI soon, which will have container building as part of the process.

**Comment by jacknugent** [comment]
> General comment (already communicated privately) -- I cannot build the frontend container. I plan on adding CI soon, which will have container building as part of the process.

File: frontend/src/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                            <option value="PA">Pennsylvania</option>
+                            <option value="RI">Rhode Island</option>
+                            <option value="SC">South Carolina</option>
+                            <option value="SD">South Dakota</option>
+                            <option value="TN">Tennessee</option>
+                            <option value="TX">Texas</option>
+                            <option value="UT">Utah</option>
+                            <option value="VT">Vermont</option>
+                            <option value="VA">Virginia</option>
+                            <option value="WA">Washington</option>
```
Comment (kevincianfarini) [inline frontend/src/HomePage/HomePage.tsx:106]: nitpick: This options list doesn't include "psuedo" states like Washington, DC and Puerto Rico and Guam. A full list of states/territories that have US congress representation can be found [here](https://www.congress.gov/members?q=%7B%22congress%22%3A117%7D). 

Look on the left hand panel for the "Members by US State or Territory" pane.

Comment (jacknugent) [reply frontend/src/HomePage/HomePage.tsx:111]: Added missing ones

**Review by kevincianfarini — APPROVED** [review]

**Pull request merged**